### PR TITLE
Gitbook Plugin for Seam Components

### PR DESCRIPTION
**Pull request by seveibar**
- test html injection
- introduce gitbook plugin for seam components

### CODE REVIEW

**Comment by sybohy** [comment]
pretty neat that this works

File: gitbook-plugin/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      <style>
+        html, body {
+          margin: 0;
+          padding: 0;
+        }
+      </style>
+      <body>
+        <script
+          type="module"
+          src="https://react.seam.co/v/1.33.3/dist/elements.js"
```
Comment (razor-x) [inline gitbook-plugin/src/index.tsx:29]:
```suggestion
          src="https://react.seam.co/v/1.42.1/dist/elements.js"
```

File: gitbook-plugin/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      <style>
+        html, body {
+          margin: 0;
+          padding: 0;
+        }
+      </style>
+      <body>
+        <script
+          type="module"
+          src="https://react.seam.co/v/1.42.1/dist/elements.js"
```
Comment (razor-x) [inline gitbook-plugin/src/index.tsx:29]: @seveibar Anyway we can hoist the version number up somewhere easy to find and bump? I'm never gonna remember where this is without bookmarking this comment 🙃

Comment (seveibar) [reply gitbook-plugin/src/index.tsx:29]: i wouldn't mind statically linking to latest but I think that doesn't currently work. I think ideally we would have this number maintained by stalebot-esque bot stuff but dogfooding latest on docs seems ok to me, wdyt

Comment (razor-x) [reply gitbook-plugin/src/index.tsx:29]: Limit to latest major version seems reasonable.

**Pull request merged**